src/stats.php

```diff
@@ -118,7 +118,7 @@ function executeContributionGraphRequests(string $user, array $years): array
  * @param string $user GitHub username to get graphs for
  * @return array<stdClass> List of contribution graph response objects
  */
-function getContributionGraphs(string $user): array
+function getContributionGraphs(string $user, int $overrideMinYear=null): array
```

Comment:
``?int` suggests it can be int or null

```suggestion
function getContributionGraphs(string $user, ?int $overrideMinYear=null): array
```

It should also be documented on line 119 above

```php
 * @param int|null $overrideMinYear Override the minimum year to get graphs for
```